lxd/instance/drivers/driver_qemu.go

```diff
@@ -2500,16 +2502,16 @@ Restart=on-failure
 RestartSec=5s
 StartLimitInterval=60
 StartLimitBurst=10
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
With that `[Install]` gone we rely on `udev` as the only trigger. As such, I think we should update the `install.sh` script to also clean the old symlink:

```
# ll /etc/systemd/system/multi-user.target.wants/lxd-agent.service 
lrwxrwxrwx 1 root root 37 Oct 27 02:35 /etc/systemd/system/multi-user.target.wants/lxd-agent.service -> /lib/systemd/system/lxd-agent.service
```

Reply:
Good point, do you mean `lxd-agent-setup` script?

Reply:
No, you can search for `lxdConfigShareInstall` and you'll see which file I'm referring to :)

Reply:
Ahh yeah, thanks. Will add it.